receiver/sqlqueryreceiver/README.md

```diff
@@ -37,7 +37,21 @@ The configuration supports the following top-level fields:
   - [snowflake](https://github.com/snowflakedb/gosnowflake) - `username[:password]@<account_identifier>/dbname/schemaname[?param1=value&...&paramN=valueN]`
   - [sqlserver](https://github.com/microsoft/go-mssqldb) - `sqlserver://username:user_password@localhost:1433?database=db_name`
   - [tds](https://github.com/thda/tds) - `tds://username:user_password@localhost:5000/db_name`
-
+- `datasource_config` (optional): Used in place of the `datasource` string. Contains separate values that are processed separately, and then concatenated into the connection string.
+  ```yaml
+  host: localhost
+  port: 5432
+  database: mydb
```

Comment:
You've added a valid example below, I think it would be more helpful to include which options are available, which are required for the direct connection/which are optional, and what value they should be given.